dnamic_analysis/output.py

```diff
@@ -0,0 +1,323 @@
+        self._excel_object.write(self._worksheet, self._col, 1, 'Expired Local Admins Total w/ Machine Addresses', 'header')
+        count = 0
+        row = 2
+        for value in max_grouped.items():
+            for key in value:
+                if not isinstance(key, str):
+                    if isinstance(key, list):
+                        for machine in key:
+                            self._excel_object.write(self._worksheet, self._col, row, machine)
+                            count = len(key)
+                            row += 1
+                else:
+                    self._excel_object.write(self._worksheet, self._col, row, key, 'subheader')
+                    row += 1
+
+        self._col += 1
+        self._excel_object.save(self._workbook)
+
+    ##############################
+    ## Local Abandoned Accounts ##
+    ##############################
+    def local_abandoned(self,abandoned_accounts, count_accounts):
+        # Write bulk data to Excel workbook
+        data = 'Total Abandoned / Total Overall: {} / {}'.format(len(abandoned_accounts), count_accounts)
+        self._excel_object.write(self._worksheet, self._col, 0, data, 'row1')
+        self._excel_object.write(self._worksheet, self._col, 1, 'Local Abandoned Accounts', 'header')
+        row = 2
+        for username,_,_,_ in abandoned_accounts:
+            self._excel_object.write(self._worksheet, self._col, row, username)
+            row += 1
+        self._col += 1
+        self._excel_object.save(self._workbook)
+
+    ###############################
+    ## Domain Abandoned Accounts ##
+    ###############################
+    def domain_abandoned(self,abandoned_accounts, count_accounts):
+        # Write bulk data to Excel workbook
+        data = 'Total Abandoned / Total Overall: {} / {}'.format(len(abandoned_accounts), count_accounts)
+        self._excel_object.write(self._worksheet, self._col, 0, data, 'row1')
+        self._excel_object.write(self._worksheet, self._col, 1, 'Domain Abandoned Accounts', 'header')
+        row = 2
+        for username,_,_,_ in abandoned_accounts:
+            self._excel_object.write(self._worksheet, self._col, row, username)
+            row += 1
+        self._col += 1
+        self._excel_object.save(self._workbook)
+
+
+    #########################################################
+    ## Accounts w/ Multiple Machine Access - By %age Tiers ##
+    #########################################################
+    def multi_machine_accts(self,multi_machine_accts):
+        if multi_machine_accts is not False:
+            tiers = ['> 95% Access','90-95% Access','80-90% Access','70-80% Access','60-70% Access',
+                '50-60% Access','40-50% Access','30-40% Access','20-30% Access','10-20% Access',
+                '< 10% Access']
+
+            self._excel_object.write(self._worksheet, 5, 1, 'Accounts with Multiple Machine Access', 'header')
+
+            for i in range(len(multi_machine_accts)):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using enumerate instead of iterating with range and len](https://app.codacy.com/app/infamousjoeg/DNAmicAnalysis/pullRequest?prid=4572226)